src/swarm/neo/client/requests/Stats.d

```diff
@@ -107,7 +109,7 @@ public class Stats : IRequestStats
         ulong start_time )
     {
         auto end_time = MicrosecondsClock.now_us();
-        assert(start_time <= end_time);
```

Comment:
Happy New Year (and the clock readjustment)! This could be in the separate commit.

Reply:
Yeah... `verify` or `enforce`, who cares, really...
(Strictly speaking, this is even a bugfix, it was asserting on input data ;)

Reply:
I moved it to a separate commit.